dynamic-plugins.default.yaml

```diff
@@ -475,6 +475,27 @@ plugins:
                     - isKind: resource
                     - isType: kubernetes-cluster
 
+  # Group: Bulk import
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
+    disabled: true
```

Comment:
@debsmita1 no additional config is needed for the backend plugin?

Reply:
no additional config is needed for the backend plugin